src/test/java/org/sonar/scm/git/blame/FileBlamerTest.java

```diff
@@ -57,6 +57,7 @@ public class FileBlamerTest {
   public void before() {
     when(revCommit.getName()).thenReturn(ANY_COMMIT_NAME);
     PersonIdent personIdent = mock(PersonIdent.class);
+    when(revCommit.getCommitterIdent()).thenReturn(personIdent);
```

Comment:
I think it would be good to have two differents personIdent in that case, so if we use author instead of committer, the test will fail .
WDYT?

Reply:
fixed